internal/schema-registry/command_dek_describe.go

```diff
@@ -0,0 +1,75 @@
+package schemaregistry
+
+import (
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/config"
+)
+
+func (c *command) newDekDescribeCommand(cfg *config.Config) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "describe",
+		Short: "Describe a DEK.",
+		Args:  cobra.NoArgs,
+		RunE:  c.dekDescribe,
+	}
+
+	cmd.Flags().String("name", "", "Name of the KEK.")
+	cmd.Flags().String("subject", "", "Subject of the DEK.")
+	pcmd.AddAlgorithmFlag(cmd)
+	cmd.Flags().String("version", "1", "Version of the DEK.")
```

Comment:
Is there a `latest` value we could use by default instead of `1`?

Reply:
Also why do we need the --version and --algorithm flag here in the first place if a DEK is defined by a (name, subject) pair?

Reply:
a DEK defined by a kek name and a subject will have different versions, they can have different "encrypted-key-material", and algorithms.

Reply:
adding version and algorithm flag will filter them.

Reply:
So this `describe` command will print out a list? This seems a bit different than other commands.

Reply:
No it'll only print out one DEK... `client.GetDekByVersion(name, subject, version, algorithm, deleted)` this call will always have one result

Reply:
`mhe@C02DV0KVMD6T cli % confluent sr dek describe --name tester --subject sche-value --algorithm AES256_GCM --version 1
+------------------------+---------------+
| Name                   | tester        |
| Subject                | sche-value    |
| Version                | 1             |
| Algorithm              | AES256_GCM    |
| Encrypted Key Material |               |
| Key Material           |               |
| Timestamp              | 1702322981244 |
+------------------------+---------------+
mhe@C02DV0KVMD6T cli % confluent sr dek describe --name tester --subject sche-value --algorithm AES256_GCM --version 2
Error: Key 'sche-value' not found
mhe@C02DV0KVMD6T cli % confluent sr dek describe --name tester --subject sche-value --version 2                       
Error: Key 'sche-value' not found
mhe@C02DV0KVMD6T cli % confluent sr dek describe --name tester --subject sche-value --algorithm AES256_SIV --version 2
+------------------------+---------------+
| Name                   | tester        |
| Subject                | sche-value    |
| Version                | 2             |
| Algorithm              | AES256_SIV    |
| Encrypted Key Material |               |
| Key Material           |               |
| Timestamp              | 1702346517344 |
+------------------------+---------------+`

Reply:
```
mhe@C02DV0KVMD6T cli % confluent sr dek describe --name tester --subject sche-value --algorithm AES256_GCM --version 1
+------------------------+---------------+
| Name                   | tester        |
| Subject                | sche-value    |
| Version                | 1             |
| Algorithm              | AES256_GCM    |
| Encrypted Key Material |               |
| Key Material           |               |
| Timestamp              | 1702322981244 |
+------------------------+---------------+
mhe@C02DV0KVMD6T cli % confluent sr dek describe --name tester --subject sche-value --algorithm AES256_GCM --version 2
Error: Key 'sche-value' not found
mhe@C02DV0KVMD6T cli % confluent sr dek describe --name tester --subject sche-value --version 2                       
Error: Key 'sche-value' not found
mhe@C02DV0KVMD6T cli % confluent sr dek describe --name tester --subject sche-value --algorithm AES256_SIV --version 2
+------------------------+---------------+
| Name                   | tester        |
| Subject                | sche-value    |
| Version                | 2             |
| Algorithm              | AES256_SIV    |
| Encrypted Key Material |               |
| Key Material           |               |
| Timestamp              | 1702346517344 |
+------------------------+---------------+
```

Reply:
If we're only printing one DEK, why are we including filtering flags?
> adding version and algorithm flag will filter them

Reply:
Or say `specify` only one of them. (might be using `filter` wrong..). `version` is default to `1`, and algorithm is default to `AES256_GCM` am i right? @rayokota

Reply:
According to Robert's prior reply I think this makes sense. version and algorithm have their default, and a combo of kek-name, subject, version number and algorithm will specify just one single DEK>